### PR TITLE
Add service polling task

### PR DESCRIPTION
**Pull request by imsickofmaps**

### CODE REVIEW

**Comment by imsickofmaps** [comment]
@gsvr review please

File: seed_control_interface_service/settings.py [hunk 1]
```diff
@@ -142,8 +143,8 @@
 HOOK_AUTH_TOKEN = os.environ.get('HOOK_AUTH_TOKEN', 'REPLACEME')
 
 # Celery configuration options
-CELERY_RESULT_BACKEND = 'djcelery.backends.database:DatabaseBackend'
-CELERYBEAT_SCHEDULER = 'djcelery.schedulers.DatabaseScheduler'
+# CELERY_RESULT_BACKEND = 'djcelery.backends.database:DatabaseBackend'
+# CELERYBEAT_SCHEDULER = 'djcelery.schedulers.DatabaseScheduler'
```
Comment (gsvr) [inline seed_control_interface_service/settings.py]: remove?

File: seed_control_interface_service/settings.py [hunk 2]
```diff
@@ -169,6 +170,9 @@
     'seed_control_interface_service.tasks.deliver_hook_wrapper': {
         'queue': 'priority',
     },
+    'seed_control_interface_service.tasks.poll_service': {
```
Comment (gsvr) [inline seed_control_interface_service/settings.py]: I think this should be `services.tasks...`

File: services/tasks.py [hunk 3]
```diff
@@ -35,3 +43,71 @@ def deliver_hook_wrapper(target, payload, instance, hook):
...
+
+        l.info("Loading Service for healthcheck")
+        try:
+            service = Service.objects.get(id=service_id)
+            l.info("Getting health for <%s>" % (service.name))
+            status = self.get_health(service.url, service.token)
+            try:
+                result = status.json()
+                print("Service is:")
+                print(result["up"])
```
Comment (gsvr) [inline services/tasks.py]: `print`s

File: services/tasks.py [hunk 4]
```diff
@@ -35,3 +43,71 @@ def deliver_hook_wrapper(target, payload, instance, hook):
...
+                service.up = result["up"]
+                Status.objects.create(
+                    service=service,
+                    up=result["up"],
+                    result=result["result"]
+                )
+                service.save()
```
Comment (gsvr) [inline services/tasks.py]: Please place this direclty under the `service.up =` line.

**Comment by gsvr** [comment]
A few minor things, :+1: when fixed.

**Pull request merged**